### PR TITLE
Cut words in design thinking and updated empathy map grammer issues

### LINKED ISSUE #6: Empathy Map

### LINKED ISSUE #17: Describe approach from Design Thinking principles perspective

### PR DESCRIPTION
**Pull request by Aiswarya668**
Closes #17 
Closes #6 

Went in and cut the words in design thinking - I can cut a bit more if we need it but didn't want to oversimplify it too much if we end up having enough words.

The empathy maps are also adjusted - nice catch Sean it had a couple of issues

### CODE REVIEW

File: Report.md [hunk 1]
```diff
@@ -66,26 +66,26 @@ Stakeholders refer to individuals/entities that have an interest in the developm
 Design Thinking Approach
 ========================
 ## **Approach - Design Thinking**
-Design thinking is a non-linear, iterative process which seeks the understanding of users to challenge assumptions, redefine problems and create innovative solutions to prototype and test. This customer-centric approach was undertaken to address the improvements requested by the Major Travel Company for their Call Management Centre (CMC). The design thinking approach undertook consists of the execution of five phases, (1) empathize, (2) define, (3) ideate, (4) prototype and (5) test and by addressing these phases we derived customer-centric innovation. 
+Design thinking is a non-linear, iterative process which seeks the understanding of users to redefine problems and create innovative solutions to prototype and test. This customer-centric approach was undertaken to address the improvements requested by the Major Travel Company for their Call Management Centre (CMC). The design thinking approach consists of five phases, (1) empathize, (2) define, (3) ideate, (4) prototype and (5) test.
```
Comment (seant1) [inline Report.md]: Just to save words
```suggestion
Design thinking is a non-linear, iterative process which seeks the understanding of users to redefine problems and create innovative solutions to prototype and test. This customer-centric approach was undertaken to address the improvements requested by the Major Travel Company for their Call Management Centre (CMC). The design thinking approach consists of the following five phases:
```

File: Report.md [hunk 2]
```diff
@@ -66,26 +66,26 @@ Stakeholders refer to individuals/entities that have an interest in the developm
...
 
 ### Define
-Phase two, define allowed us to synthesise the knowledge obtained in the empathise stage and define the core problems to solve. Here the problem statements were directly derived from creating point of view (POV) statements for all the stakeholders identified. This stage allowed us as a team to consolidate assumptions and agree on features to be developed to solve the problem given. 
+Phase two, define consisted of synthesising the knowledge obtained in the empathise stage and defining core problems to solve. Here the problem statements were directly derived from creating point of view (POV) statements for all the stakeholders identified. This stage allowed us as a team to consolidate assumptions and agree on features to be developed to solve the problem given.
```
Comment (seant1) [inline Report.md]: Shortening...
```suggestion
Phase two consisted of synthesising the knowledge obtained in the empathise stage and defining core problems to solve. Here the problem statements were directly derived from creating point of view (POV) statements for all the stakeholders identified. This stage allowed us as a team to consolidate assumptions and agree on features to be developed to solve the problem given.
```

File: Report.md [hunk 3]
```diff
@@ -66,26 +66,26 @@ Stakeholders refer to individuals/entities that have an interest in the developm
...
+Phase three consisted of utilising user research obtained to ideate solutions to the problem statements we created in the prior stage. Brainstorming and 'worst possible idea' techniques were utilised to assist this stage. These techniques were chosen to initiate freeform group discussions and stimulate creative solutions. Ideate sessions consisted of developing “how might we? (HMW)” statements and deriving features relative to the point of view statements developed in the define stage. At the conclusion of this stage, the agreed-upon features were documented and converted to issues in git  to be developed for the prototype and testing stage.
```
Comment (seant1) [inline Report.md]:
```suggestion
Phase three consisted of utilising user research obtained to ideate solutions to the problem statements created in the prior stage. Brainstorming and 'worst possible idea' techniques were utilised. These techniques helped initiate free-form group discussions and stimulate creative solutions. Ideate sessions consisted of developing “how might we?" (HMW) statements and deriving features relative to the point of view statements developed. At the conclusion of this stage, the agreed-upon features were documented and converted to issues in GitHub to be developed for the prototype and testing stage.
```

**Review by seant1 — APPROVED** [review]
Good work. Suggestions to cut it down more (but only really by a few words).

File: Report.md [hunk 4]
```diff
@@ -66,26 +66,26 @@ Stakeholders refer to individuals/entities that have an interest in the developm
...
+Phase one consisted of empathising with stakeholders. Empathy is a crucial step in achieving a human-centred design process and was utilised to gain insight into users and their needs without imposing bias. When we create solutions, we get better solutions when we think of those who will be affected. To document these perspectives, empathy maps were utilised to visualise and share our understanding of stakeholders needs and aid future decision making. Phase one allowed us to gain a deeper understanding of the perspective of the users involved and how to shape our solution.
```
Comment (zachartmann) [inline Report.md:72]: "When we create solutions, we get better solutions when we think of those who will be affected."
Just wondering if we need this sentence. It's definitely true but maybe already assumed

**Review by zachartmann — APPROVED** [review]
Awesome, well done - just one thought

File: Report.md (hunk 4 above)
Comment (Aiswarya668) [reply Report.md:72]: Yea I was thinking about getting rid of that part - I'll take it out if we need the words :) was thinking of revisiting this later as well just to scrape some more words where I can

**Pull request merged**